qmf/src/libraries/qmfclient/qmailmessage.cpp

```diff
@@ -1047,6 +1047,9 @@ namespace findBody
         case QMailMessagePart::MultipartNone:
             return inMultipartNone(part, ctx);
 
+        case QMailMessagePart::MultipartMixed:
+            return inMultipartMixed(part, ctx);
+
```

Comment:
This is the default already below in line 1061, but is behind a warning, so adding here as well to make is more clear.